### DISCUSSION TITLE
How to use external package in vsketch project?

### DISCUSSION THREAD

**Discussion by oragamiman01** [abey79/vsketch#386] [Q&A] [answered]
Hello, I don't know what I need to do to make a package like opencv or scikit-image work in a vsketch project. I am able to use those packages when I import vsk as a normal package in a python file or notebook, but not when I import them in a sketch project. It's not a super big issue since I can get around it that way but it would be nice if I could use those packages with the vpype viewer. 

Thanks for creating this tool! I really enjoy using it.

**Comment by abey79** [comment, marked as answer]
I assume you installed `vsk` using pipx, as recommended. If so, when executing your sketches with `vsk run`, the pipx managed virtual environment of `vsk` is used to actually run the sketch. So this is where you need to install any dependencies you need. Luckily, pipx has support for this:
```
pipx inject vsketch python-opencv 
```
